css/styles.css

```
@@ -177,24 +177,24 @@ body {
   border-radius: 117px;
 }
 
-.profile__name-date {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin-bottom: 2px;
+.profile__image--responsive {

```

Essa class nao necessariamente é uma modificador. Ela apenas teria um comportamento diferente no Mobil/Tablet.

Poderia renomear para algo especifico do responsivo, e não sendo um modificador.
```suggestion
.profile__image-desktop {
```